resolos/archive.py

```diff
@@ -253,24 +267,38 @@ def load_archive_file(input_filename: str, files_path):
                     execute_local_conda_command(
                         f"install -y --name {new_env_name} --file {explicit_packages_path}"
                     )
+                    install_pip_packages(
+                        new_env_name, requirements_path, resolos_version
+                    )
                     project_settings["env_name"] = new_env_name
                     if old_env_name:
                         execute_local_conda_command(
                             f"remove -y --name {old_env_name} --all"
                         )
                     pdc.write(project_settings)
                 except Exception as ex:
-                    clog.info(
-                        f"Failed to load conda env using explicit packages list, "
-                        f"will use now conda-pack..."
-                    )
-                    clog.debug(f"The error was:\n\n{ex}\n\n")
-                    extract_file(tar, PACK_NAME, pack_absolute_path)
-                    run_shell_cmd(
-                        f"mkdir -p ~/.resolos/envs/{new_env_name} && "
-                        f"tar -xzf {pack_absolute_path} -C ~/.resolos/envs/{new_env_name} && "
-                        f"source ~/.resolos/envs/{new_env_name}/bin/activate && "
-                        f"conda-unpack",
+                    if Path(pack_absolute_path).exists():
+                        clog.info(
+                            f"Failed to load conda env using explicit packages list, "
```

Comment:
Let's separate this message into 2 parts:
1. The message "Failed to load conda env using explicit packages list" should always appear in this codepath
2. Then, the next message is either "Trying installing fromconda pack" or "Trying installing from explicitly installed packages list " depending whether the archive has the packed binaries